tools/nl/embeddings/data/curated_input/main/main_topics.csv

```diff
@@ -7,15 +7,14 @@ dc/topic/Jobs,Jobs,Jobs,, Careers; Professions; Work Occupations; Employment Pos
 dc/topic/ProjectedClimateExtremes,Projected Climate Extremes,Projected Climate Extremes,,Climate Extremes;
 dc/topic/ClimateChange,Impact of Climate Change,Climate Change impact,,Climate Change; Global Warming
-dc/topic/Income,Income,Income from Earnings,,Earnings; Salary; pay; compensation; wealth; what are the wealthy neighborhoods; monetary status
+dc/topic/Income,Income,Income from Earnings,,Earnings; Salary; pay; compensation
 dc/topic/WorkCommute,Work Commute,journey to work; travel to work; commute mode,,Work commute;
 dc/topic/AgricultureEmissionsByGas,Emissions from agriculture,Emissions from agriculture,,Amount of gases emitted by agricultural sector;Total quantity of gases released by the farming industry; The volume of emissions produced by agriculture; Quantity of greenhouse gases emitted by the agricultural sector; Total amount of gas emissions from farming practices; The sum of all gases released by the agricultural industry;
 dc/topic/FossilFuelOperationsEmissionsByGas,Emissions from fossil fuel operations,Emissions from fossil fuel operations,,Amount of gases emitted in fossil fuel operations;Total quantity of gases released by fossil fuel operations; The volume of emissions produced by the use of fossil fuels; Quantity of greenhouse gases emitted by the burning of fossil fuels; Total amount of gas emissions from fossil fuel operations; The sum of all gases released by the use of fossil fuels;
 dc/topic/TransportationEmissionsByGas,Emissions from transportation,Emissions from transportation,,Amount of gases emitted by the transportation sector;Total quantity of gases released by the transportation sector; The volume of emissions produced by transportation activities; Quantity of greenhouse gases emitted by vehicles and transportation systems; Total amount of gas emissions from the transportation industry; The sum of all gases released by transportation activities;
 dc/topic/ManufacturingEmissionsByGas,Emissions from manufacturing,Emissions from manufacturing,,Amount of gases emitted by the manufacturing sector;Total quantity of gases released by the manufacturing industry; The volume of emissions produced by manufacturing activities; Quantity of greenhouse gases emitted by factories and production processes; Total amount of gas emissions from the manufacturing sector; The sum of all gases released by manufacturing activities;
 dc/topic/PowerEmissionsByGas,Emissions from power,Emissions from power,,Amount of gases emitted by the power sector;Total quantity of gases released by the power industry; The volume of emissions produced by electricity generation activities; Quantity of greenhouse gases emitted by power plants and energy production processes; Total amount of gas emissions from the power sector; The sum of all gases released by power generation activities;
 dc/topic/Literacy,,Literacy,,
-dc/topic/SDG_1,,,,End poverty; End poverty in all its forms; SDG end poverty; SDG poverty;
```

Comment:
can you check that the electrification demo looks ok removing this? we kept this previously because the actual dc/topic/sdg_1 caused problems for that demo: https://github.com/datacommonsorg/website/blob/master/server/lib/nl/common/topic.py#L67-L78

Reply:
The charts shown in local seems the same as that of autopush

Reply:
Also confirmed that there is no diff for e2e_electrification_demo test